.bazelrc

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 # Force the use of Clang for all builds.
-build --action_env=CC=clang
-build --action_env=CXX=clang++
+build --action_env=CC=clang-10
```

Comment:
Does this make these rules less portable? i.e. on distros that use a different clang version. 

Or is this test/CI only?

Reply:
Ah, this only applies for this repository's `WORKSPACE`, which is used for rule development, CI, and testing. The projects using these rules have their own `WORKSPACE` files and they set up their own C++ toolchain.